cfgov/unprocessed/apps/youth-employment-success/js/views/review/choice.js

```diff
@@ -0,0 +1,118 @@
+import { checkDom, setInitFlag } from '../../../../../js/modules/util/atomic-helpers';
+import { updateRouteChoiceAction } from '../../reducers/choice-reducer';
+import { toArray } from '../../util';
+import transportationMap from '../../data/transportation-map';
+import inputView from '../input';
+
+const CLASSES = Object.freeze( {
+  CONTAINER: 'js-yes-review-choice',
+  REVIEW_PLAN: 'js-yes-plans-review',
+  BUTTON: 'js-yes-choice-finalize',
+  CHOICE: 'js-yes-route-selection'
+} );
+
+function reviewChoiceView( element, { store } ) {
+  const _dom = checkDom( element, CLASSES.CONTAINER );
+  const _reviewPlanEl = _dom.querySelector( `.${ CLASSES.REVIEW_PLAN }` );
+  const _choiceBtnEls = toArray(
+    _dom.querySelectorAll( `.${ CLASSES.CHOICE }` )
+  );
+  const _reviewBtnEl = _dom.querySelector( `.${ CLASSES.BUTTON }` );
+
+  function _enableChoices() {
+    _choiceBtnEls.forEach( el => el.querySelector( 'input' ).setAttribute( 'disabled', false )
+    );
+  }
+
+  function _disableChoices() {
+    _choiceBtnEls.forEach( el => el.querySelector( 'input' ).setAttribute( 'disabled', 'disabled' )
```

Comment:
Could we store the inputs upfront so the `el` isn't queried when both enabling and disabling the choices?

Reply:
The linter weirdly made this harder to read 😦